mesonbuild/modules/gnome.py

```diff
@@ -1689,5 +1689,93 @@ def generate_vapi(self, state, args, kwargs):
         created_values.append(rv)
         return ModuleReturnValue(rv, created_values)
 
+    @permittedKwargs({'package_name', 'package_version', 'packages',
+                      'doclet', 'doclet_args', 'vapi_dirs', 'install', 'install_dir'})
+    @FeatureNew('gnome.valadoc', '0.55.0')
+    def valadoc(self, state, args, kwargs):
+        build_dir = os.path.join(state.environment.get_build_dir(), state.subdir)
+        source_dir = os.path.join(state.environment.get_source_dir(), state.subdir)
+
+        if not args:
+            raise MesonException('valadoc takes at least one positional argument')
+
+        # Build the command line
+        cmd = [ self.interpreter.find_program_impl('valadoc') ]
```

Comment:
I don't see the rest of the gnome module using spaces around `[` and `]` like this, and we don't do it anywhere else in the code base.

Reply:
Fixed!